scripts/fetch_licenses.js

```diff
@@ -4,13 +4,11 @@
 // single file that provides the most-interesting (to us) information from both.
 
 import fs from 'node:fs/promises';
-import path from 'node:path';
 
-const __dirname = new URL('.', import.meta.url).pathname;
-const __filename = new URL(import.meta.url).pathname;
-
-const LICENSE_FILE = path.join(__dirname, '../lib/licenses.ts');
-const AUTOGENERATE_TOKEN = '--- AUTOGENERATED FROM HERE DOWN ---';
+const LICENSE_FILE = new URL(
+  '../lib/licenses_autogenerated.json',
```

Comment:
Change this to "../data/licenses.json"?  That'll keep generated [json] files out of the [source] `lib` dir, and I could see having more fetched / generated files like this. E.g. for #254

Reply:
Sure. However as long as the data lives on GitHub and you can fetch it directly.